docs/user-guide/admin-client/consumer.rst

```diff
@@ -76,7 +76,7 @@ bindings are waiting for that remote action to take place.
     Available Arguments:
 
       --fields   - comma-separated list of consumer fields; if specified only the
-                   given fields will be displayed. Example: "id,display_name"
+                   given fields will be displayed. Example: "id,display_name".
```

Comment:
Is the Example an example of the input or the output? Since it comes after the explanation of what is displayed it looks like you are giving an example of the output. If you are giving an example of the input, can the example be moved to after the description of the input. Also id,display_name could be escaped using backticks. Maybe it would read like this:

```
comma-separated list of consumer fields. For example `id,display_name` . If specified only the given fields will be displayed.
```